### ISSUE TITLE
Allow not checking @flow in comments

### ISSUE THREAD

**Issue by hanayashiki** [dalcib/esbuild-plugin-flow#2]
Some .flow files are not annotated but needs transformation, like react-native. So it would be nice if we can add a `{ force:  boolean }`!

**Comment by dalcib** [comment]
Ok, thanks.
Fixed on v0.3.2

**Comment by hanayashiki** [comment]
> Ok, thanks.
> Fixed on v0.3.2

i meant that should really be fixed by flow remove types

The current workaround does not take syntax into account

**Comment by hanayashiki** [comment]
when i have time i'd like to give them a PR or something to fix the problem. Anyway, flow is dead, don't expect they'll merge it

**Comment by carlgieringer** [comment]
Hi, nanayahsiki, can you please explain what this syntax means? `static +x: "literal" = "literal"`? I am not familiar with that syntax involving the `+` sign.  Does the following code show the same syntax?

```
class MyClass
  static +myStaticMember: "SomeLiteralValue" = "SomeLiteralValue";
}
```

What does the `+` in front of the identifier do? Thanks!

**Comment by hanayashiki** [comment]
> Hi, nanayahsiki, can you please explain what this syntax means? `static +x: "literal" = "literal"`? I am not familiar with that syntax involving the `+` sign. Does the following code show the same syntax?
> 
> ```
> class MyClass
>   static +myStaticMember: "SomeLiteralValue" = "SomeLiteralValue";
> }
> ```
> 
> What does the `+` in front of the identifier do? Thanks!

yes, that's what i meant. It is just a flow syntax. I don't use flow but it probably means 'public'. 

`flow-remove-types` itself does not properly handle this syntax, it leaves it there.
